example/app-router/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "app-router",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "dotenv": "^16.4.5",
+    "next": "14.2.3",
```

Comment:
add the following dependecy
`
    "imagekit-next": "file:../../imagekit-next-1.0.0.tgz",
`

Reply:
done